client/src/styles/Footer.css

```diff
@@ -11,6 +11,7 @@ footer {
     font-family: var(--secondary-font);
     text-align: center;
     height: 6.5rem;
+    position: absolute;
```

Comment:
no position absolute here! the footer is in normal position on all pages, (class "footer-normal-position), except on Home page where is has the class name "footer-position-fixed" to ensure that it is shown above the map.

Reply:
ok

Reply:
yes please

Reply:
done